oak_functions/loader/src/server.rs

```diff
@@ -816,15 +831,26 @@ fn oak_functions_resolve_func(field_name: &str) -> Option<(usize, wasmi::Signatu
 /// `wasmi` specific result type `Result<Option<wasmi::RuntimeValue>, wasmi::Trap>`, mapping:
 /// - `Ok(())` to `Ok(Some(OakStatus::Ok))`
 /// - `Err(x)` to `Ok(Some(x))`
-fn map_host_errors(
-    result: Result<(), OakStatus>,
-) -> Result<Option<wasmi::RuntimeValue>, wasmi::Trap> {
+fn oak_to_wasmi(result: Result<(), OakStatus>) -> Result<Option<wasmi::RuntimeValue>, wasmi::Trap> {
```

Comment:
Could this be `oak_status_to_wasmi_value`?

Reply:
Are you suggesting to rename the function? 

Happy to, because I am not too convinced of as it is right now, but not too sure how. I think, we are not really mapping an OakStatus but somehow between "results in Oak and Wasmi", no?

Reply:
I agree that `map_host_errors` is a bit misleading, as it is not mapping only the errors. But also we are not mapping `oak` :) Perhaps `oak_result_to_wasmi_result` or `map_host_result`, or `map_host_result_to_wasmi`.